INSTALL.md

```diff
@@ -1,44 +1,56 @@
-### [Visual Studio](https://www.visualstudio.com/)
```

Comment:
This title is important and can't be removed, without that it will not render the website correctly.